<launch>
  <include file="$(find slaw_registration)/launch/icp.launch" />
  <include file="$(find slaw_navigation)/launch/slaw_global_navigation.launch" />
  <include file="$(find slaw_arm_navigation)/launch/slaw_arm_navigation.launch" />
  <include file="$(find slaw_object_detection)/launch/usb_cam.launch" />
  <include file="$(find slaw_vision)/launch/usb_cam.launch" />

  <node pkg="slaw_navigation" name="odom_move" type="odom_move_action.py" />
</launch>